cli/internal/packagemanager/pnpm.go

```diff
@@ -28,7 +29,8 @@ var nodejsPnpm = PackageManager{
 	// We are allowed to use nil here because ArgSeparator already has a type, so it's a typed nil,
 	// This could just as easily be []string{}, but the style guide says to prefer
 	// nil for empty slices.
-	ArgSeparator: nil,
+	ArgSeparator:  nil,
+	WorkspacePath: "pnpm-workspace.yaml",
```

Comment:
`WorkspacePath` sounds like an individual workspace _thing_.

How about something that would match this regex?
`Workspace(Definition|Configuration)(Path|File)`